data/water/wetlands/index.html

```diff
@@ -46,31 +34,14 @@ <h3 id="Wetlands">
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
     <img class="inline-text-left outline" src="{{ "/images/wetlands2.png" | prepend: site.baseurl }}" alt="Wetlands" />
-    <p>Water.Wetlands represents wetland areas in Utah compiled from NWI, USFS, and UGS sources. In addition to the Cowardin fields, data has been attributed by the Utah Geological Survey to provide functional and descriptive codes associated with each wetland class. This data set is actively being maintained by the Utah Geologic Survey.</p>
+    <p>Water.Wetlands is derived from the U.S. Fish and Wildlife Service’s [National Wetlands Inventory](https://www.fws.gov/wetlands/) (NWI). NWI attributes wetlands with codes from the Cowardin Classification System; codes can be interpreted [online](https://www.fws.gov/wetlands/data/wetland-codes.html). Cowardin classes in this derived layer have been split into separate System, Class, and Water Regime fields. Wetlands spatial data were produced from a combination of aerial imagery examination and on-the-ground assessment and are not meant to be used as the basis for a jurisdictional wetland delineation.</p> 
```

Comment:
nah, markdown in html like this won't work.

```suggestion
    <p>Water.Wetlands is derived from the U.S. Fish and Wildlife Service’s <a href="https://www.fws.gov/wetlands/">National Wetlands Inventory</a> (NWI). NWI attributes wetlands with codes from the Cowardin Classification System; codes can be interpreted <a href="https://www.fws.gov/wetlands/data/wetland-codes.html">online</a>. Cowardin classes in this derived layer have been split into separate System, Class, and Water Regime fields. Wetlands spatial data were produced from a combination of aerial imagery examination and on-the-ground assessment and are not meant to be used as the basis for a jurisdictional wetland delineation.</p> 
```